static/js/components/VideoPlayer.js

```diff
@@ -247,20 +251,21 @@ export default class VideoPlayer extends React.Component<*, void> {
         "FullscreenToggle"
       ).prototype.handleClick = toggleFullscreen
     }
-    this.percentTracked = []
+    this.lastMinuteTracked = -1
```

Comment:
Can you use `null` here instead to make it more explicit when the value is not set?

Reply:
✅